zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -17,10 +17,28 @@
  */
 package org.apache.zookeeper.common;
 
+import org.apache.zookeeper.common.X509Exception.KeyManagerException;
```

Comment:
nit: revert import reorder? Or fix your import organizer to do java.* -> javax.* -> everything else